### DISCUSSION TITLE
how to set a valuator counter as a "simplecounter" ?

### DISCUSSION THREAD

**Discussion by ffred** [fltk-rs/fltk-rs#1247] [General]
Hi,
I'm making my first steps with fltk-rs (a big thank you to MoAlyousef for this great work !) and just set a counter valuator, but can't find how to set it as a "simplecounter" ?

here's my actual counter code : 
```
    let mut cnter = valuator::Counter::default().with_size(80, 36).below_of(&*table, 20);
    cnter.set_minimum(2.);
    cnter.set_maximum(99.);
    cnter.set_step(1., 1);
    cnter.set_value(2.);
```
thanks for your help..
ffred

**Comment by MoAlyousef** [comment]
Hi
Thank you for your kind words. You can set the counter's type like so:
```rust
let mut cnter = valuator::Counter::default().with_size(80, 36).with_type(valuator::CounterType::Simple).below_of(&*table, 20);
```

**Comment by ffred** [comment]
oh nice.. thank you !!
